core/src/dird/ua_select.cc

```diff
@@ -1124,6 +1176,9 @@ int DoPrompt(UaContext* ua,
   PoolMem pmsg(PM_MESSAGE);
   BareosSocket* user = ua->UA_sock;
 
+  int windowsize = 80;
```

Comment:
when reading `windowssize` my first idea is with and height. However, your variable just contains the width, so maybe `window_width` might be a better name.